doc/sfp-cmis/cmis.md

```diff
@@ -0,0 +1,308 @@
+# Feature Name
+QSFPDD CMIS support for SONiC
```

Comment:
nit: All references to QSFP-DD should probably be removed (just CMIS support for SONiC) or include OSFP since both use the CMIS interface